community-operators/okd-windows-machine-config-operator/0.0.1/okd-windows-machine-config-operator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,328 @@
+    OKD, remove all Windows machinesets and uninstall this operator. Once the OKD upgrade process has completed,
+    you may reinstall the operator, and recreate all Windows machinesets.
+
+    ## Documentation
+
+    ### Introduction
+    The Windows Machine Config Operator configures Windows Machines into nodes, enabling Windows container workloads to
+    be ran within OKD clusters. The operator is configured to watch for Machines with a `machine.openshift.io/os-id: Windows`
+    label. The way a user will initiate the process is by creating a MachineSet with this label specifying an Windows
+    image that has the container runtime installed. The operator will do all the necessary steps to configure the underlying
+    VM so that it can join the cluster as a worker node.
+
+    ### Pre-requisites
+    * OKD 4.6 cluster configured with hybrid OVN Kubernetes networking
+
+    ### Usage
+    Once the `windows-machine-config-operator` namespace has been created, a secret must be created containing the private key
+    that which will be used to access the Windows VMs. The private key should be in PEM encoded RSA format.
+    ```
+    # Change paths as necessary
+    oc create secret generic cloud-private-key --from-file=private-key.pem=$HOME/.ssh/$keyname -n windows-machine-config-operator
+    ```
+
+    Below is the example of an AWS Windows MachineSet which can create Windows Machines that the WMCO can react upon.
+    Please note that the windows-user-data secret will be created by the WMCO lazily when it is configuring the first
+    Windows Machine. After that, the windows-user-data will be available for the subsequent MachineSets to be consumed.
+    It might take around 10 minutes for the Windows VM to be configured so that it joins the cluster. Please note that
+    the MachineSet should have following labels:
+    * `machine.openshift.io/os-id: Windows`
+    * `machine.openshift.io/cluster-api-machine-role: worker`
+    * `machine.openshift.io/cluster-api-machine-type: worker`
+    The following label, needs to be added to the Machine spec within the MachineSet spec:
+    * `node-role.kubernetes.io/worker: ""`
+
+    Not having these labels will result in the Windows node not being marked as a worker.
+
+    `<windows_container_ami>` should be replaced with the AMI ID of a Windows image with a container run-time installed.
+    <infrastructureID> should be replaced with the output of:
+    `oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster`
+
+    `<region>` should be replaced with a valid AWS region like `us-east-1`.
+    `<zone>` should be replaced with a valid AWS availability zone like `us-east-1a`.
+
+    ```
+    apiVersion: machine.openshift.io/v1beta1
+    kind: MachineSet
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructureID>
+      name: <infrastructureID>-worker-<zone>
```

Comment:
`name: <infrastructureID>-windows-worker-<zone>` otherwise this maps to an existing Linux. Ref: https://github.com/openshift/windows-machine-config-operator/pull/117

Reply:
The same change needs to be made to the labels.

Reply:
Shouldn't the name be unique